test/unit/app/controllers/permissions/mocks.js

```diff
@@ -162,10 +163,19 @@ const PERM_NAMES = {
   does_not_exist: 'does_not_exist',
 }
 
-const ACCOUNT_ARRAYS = {
-  a: [ ...keyringAccounts.slice(0, 3) ],
-  b: [keyringAccounts[0]],
-  c: [keyringAccounts[1]],
+const ACCOUNTS = {
+  a: {
+    permitted: [...keyringAccounts.slice(0, 3)],
```

Comment:
nit: `slice` returns a new array, so the spread is unnecessary

Reply:
fixed.